provision/setup.py

```diff
@@ -1,9 +1,16 @@
+import sys
+sys.dont_write_bytecode = True
+
 from setuptools import setup, find_packages
+import os 
+file_dir = os.path.dirname(__file__)
```

Comment:
This is good. Hopefully, It will pass rpm and deb job this time with this chance.